cli/cmd/volume.go

```diff
@@ -325,3 +331,27 @@ var volumeStatusCmd = &cobra.Command{
 		fmt.Println("STATUS:", volname)
 	},
 }
+
+var volumeExpandCmd = &cobra.Command{
+	Use:   "add-brick",
+	Short: helpVolumeExpandCmd,
+	Run: func(cmd *cobra.Command, args []string) {
+		validateNArgs(cmd, 2, 0)
+		volname := cmd.Flags().Args()[0]
+		bricks, err := bricksAsUUID(cmd.Flags().Args()[1:])
+		if err != nil {
+			log.WithField("volume", volname).Println("addition of brick failed")
+			failure(fmt.Sprintf("Error getting brick UUIDs: %s", err.Error()), 1)
+		}
+		vol, err := client.VolumeExpand(volname, api.VolExpandReq{
+			ReplicaCount: flagCreateCmdReplicaCount,
+			Bricks:       bricks, // string of format <UUID>:<path>
+		})
+		if err != nil {
+			log.WithField("volume", volname).Println("volume expansion failed")
+			failure(fmt.Sprintf("Addition of brick failed with %s", err.Error()), 1)
+		}
+		fmt.Printf("%s Volume expanded successfully\n", vol.Name)
+		fmt.Println("Volume ID: ", vol.ID)
```

Comment:
Printing volume id is useful only during volume create. You can remove this line.